--- a/include/quaternion.h
+++ b/include/quaternion.h
@@ -29,15 +29,14 @@
 #ifndef QUATERNIONS_QUATERNION_H
 #define QUATERNIONS_QUATERNION_H
 
-#include <math.h> // for atan2, which handles signs for us properly
-
+#include <cmath> // for atan2, which handles signs for us properly
+#include <cstdint>
 #include <limits>
 #include <type_traits>
 #include <array>
 #include <complex>
 #include <iterator>
 #include <assert.h>
-#include <boost/mpl/bool.hpp>
 
 #include "quaternion_utils.h"
 
